JSDemos/Demos/Validation/Overview/description.md

```diff
@@ -1,9 +1,34 @@
 
+- [compare](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/)    
+A validation rule that demands that a validated editor has a value that is equal to a specified expression. For this rule type, implement the [comparisonTarget](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CompareRule/#comparisonTarget) function to specify the value that component compares with a validated value.
 
-Note that the _&laquo;Register&raquo;_ button here does not implement a&nbsp;usual **onClick** event handler. Instead, it&nbsp;has the **useSubmitBehavior** property set to _true_. This setting tells the button to&nbsp;validate and submit the HTML form in&nbsp;which it&nbsp;is&nbsp;nested, with no&nbsp;further configuration required.
\ No newline at end of file
+- [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/)    
+A validation rule that demands that the validated field matches a specified [pattern](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/PatternRule/#pattern). 
+
+- [stringLength](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/)   
+A validation rule that demands the target value length be within the range of specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/StringLengthRule/#max). This property accepts only string values.
+
+- [range](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/)  
+A validation rule that demands the target value length be within the range of specified [minimum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#min) and [maximum](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RangeRule/#max). This property accepts only date-time or numeric values.  
+
+- [numeric](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/NumericRule/)    
+A validation rule that demands that the validated field has a numeric value.
+
+- [custom](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/CustomRule/)    
+A rule with custom validation logic. Implement the [validationCallback](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/AsyncRule/#validationCallback) function to validate the target value.
+
+No matter the type, all validation rules allow you to specify an error [message](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#message) for a component. If your application implements a validation group, you can use the [ValidationSummary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place. In this demo, click the **Register** button to see a validation summary.
```

Comment:
> No matter the type

This isn't the best way to start a paragraph in documentation... Moreover, I believe we can go by without it at all.

> you can use the [ValidationSummary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place.

ValidationSummary - an API member used as ordinary words in a sentence... This isn't the way to do things properly.

```suggestion
All validation rule types allow you to specify an error [message](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#message) for a component. If your application implements a validation group, you can use a [validation summary](/Documentation/ApiReference/UI_Components/dxValidationSummary/) to display all validation errors in one place. In this demo, click the **Register** button to see a validation summary.
```